src/RealtimeEffectPanel.cpp

```diff
@@ -903,12 +904,30 @@ class RealtimeEffectListWindow : public wxScrolledWindow
    }
 };
 
+
+struct RealtimeEffectPanel::PrefsListenerHelper : PrefsListener
```

Comment:
This commit is insufficient to update langauges in (e.g.) the dialog for Reverb.  It only fixes the dialog title, not the static texts and label buttons.  Destroying and recreating the dialog may do it.  That is I think `RealtimeEffectStateUI::Hide` then `RealtimeEffectStateUI::Show`